src/servers/graphql-old-server.ts

```diff
@@ -189,9 +188,13 @@ const resolvers = {
     },
     usernameExists: async (_, { username }) => usernameExists({ username }),
     getUserDetails: async (_, { uid }) => User.findOne({ _id: uid }),
-    noauthUpdatePendingInvoice: async (_, { hash, username }, { logger }) => {
-      const wallet = await getWalletFromUsername({ username, logger })
-      return wallet.updatePendingInvoice({ hash })
+    noauthUpdatePendingInvoice: async (_, { hash }, { logger }) => {
```

Comment:
@jrbuhl93 @dolcalmi @samerbuna we'd need to just manually test that this behaves the same way in the places where  this endpoint is currently used.

Same for the [`updatePendingInvoice` endpoint below](https://github.com/GaloyMoney/galoy/pull/509/files#diff-c5d23ef164ed6aa31783566254b690eec1b58fc6c80fc1e851020f2eaf256d8bR269)